Userland/Applications/PixelPaint/Image.cpp

```diff
@@ -528,6 +528,47 @@ void Image::crop(Gfx::IntRect const& cropped_rect)
     did_change_rect(cropped_rect);
 }
 
+Optional<Gfx::IntRect> Image::bounding_rect() const
+{
+    if (m_layers.is_empty())
+        return {};
+
+    Optional<int> min_x;
+    Optional<int> min_y;
+    Optional<int> max_x;
+    Optional<int> max_y;
```

Comment:
Let's use two `IntPoint`s for these instead

Reply:
What do you think about this?
```
Optional<Gfx::IntRect> Image::bounding_rect() const
{
    if (m_layers.is_empty())
        return {};

    Optional<Gfx::IntRect> rect;

    for (auto const& layer : m_layers)
        rect = rect.has_value() ? rect->united(layer.relative_rect()) : layer.relative_rect();

    return rect;
}
```